### PR TITLE
#30 Add drifter section

### PR DESCRIPTION
**Pull request by kromash**
Drift analysis is enabled for pairs of explainers provided in list e.g. `modelDown(explainer_lm, list(explainer_glm_old, explainer_glm_new) ...`

### CODE REVIEW

File: R/modelDown.R [hunk 1]
```diff
@@ -36,24 +36,27 @@
...
 #'
-#' modelDown::modelDown(explainer_ranger, explainer_glm) #all defaults
+#' modelDown::modelDown(explainer_ranger, list(explainer_glm1, explainer_glm2)) #all defaults
 #'
-#' modelDown::modelDown(explainer_glm,
-#'   modules = c("auditor", "model_performance", "variable_importance",
-#'               "variable_response"),
+#' modelDown::modelDown(list(explainer_glm1, explainer_glm2)
+#'   modules = c("auditor", "drifter", "model_performance", "variable_importance",
+#'               "variable_response", "prediction_breakdown"),
```
Comment (magda-tatarynowicz) [inline R/modelDown.R]: prediction_breakdown has already been removed

File: inst/extdata/modules/drifter/styles.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.data-container {
+  padding: 0px 20px;
+}
+
+.plot {
```
Comment (magda-tatarynowicz) [inline inst/extdata/modules/drifter/styles.css]: I think that latest changes have width set to 60%

**Review by pbiecek — COMMENTED** [review]
For some reason I see only an empty page for drifter, when running
```
library("titanic")
titanic <- titanic_train[,c("Survived", "Pclass", "Sex", "Age", "SibSp", "Parch", "Fare", "Embarked")]
titanic$Survived <- factor(titanic$Survived)
titanic$Sex <- factor(titanic$Sex)
titanic$Embarked <- factor(titanic$Embarked)
titanic <- na.omit(titanic)
titanic <- titanic[titanic$Embarked != "",]
titanic$Embarked <- factor(titanic$Embarked)
head(titanic)

library("randomForest")
rf_model <- randomForest(Survived == "1" ~ .,  data = titanic)
rf_model

library("DALEX")
rf_explain <- explain(rf_model, data = titanic[,-1],
                      y = titanic$Survived == "1", label = "RF")

modelDown::modelDown(rf_explain, device = "svg",
                     remote_repository_path = "MI2DataLab/modelDown_example",
                     output_folder = "../modelDown_examples_titanic_one")
```

File: R/modelDown.R (hunk 1 above)
Comment (kromash) [reply R/modelDown.R]: Removed

File: inst/extdata/modules/drifter/styles.css (hunk 2 above)
Comment (kromash) [reply inst/extdata/modules/drifter/styles.css]: Removed that plot css because it was not used

**Comment by kromash** [comment]
@pbiecek I fixed problem with empty drifter page

**Review by magda-tatarynowicz — APPROVED** [review]

**Pull request merged**